mozci/console/commands/push.py

```diff
@@ -365,32 +422,20 @@ def handle(self) -> None:
             self.option("rev"),
         )
 
+        option_names = [
+            "intermittent-confidence-threshold",
+            "real-confidence-threshold",
+            "use-possible-regressions",
+            "unknown-from-regressions",
+            "consider-children-pushes-configs",
+            "cross-config-counts",
+            "consistent-failures-counts",
+        ]
```

Comment:
Nit: we could use inspect.signature here too to avoid listing them all

Reply:
Change pushed :smile: